client-js/evomaster-client-js/package-lock.json

```diff
@@ -1,8 +1,6436 @@
 {
```

Comment:
why was this file modified?

Reply:
if there was no specific reason, revert it

Reply:
seems like file got updated when the new branch created. change is reverted